packages/styling/src/styles/theme.ts

```diff
@@ -60,11 +62,12 @@ export function loadTheme(theme: IPartialTheme): ITheme {
  * Creates a custom theme definition which can be used with the Customizer.
  */
 export function createTheme(theme: IPartialTheme): ITheme {
-  let newPalette = { ...DefaultPalette, ...theme.palette };
 
-  if (!theme.palette || !theme.palette.accent) {
-    newPalette.accent = newPalette.themePrimary;
-  }
+  let newPalette = { ...DefaultPalette };
```

Comment:
no, this needs to be:
let newPalette = { ...DefaultPalette, ...theme };
So that we pick up overrides of the default colors, like if they customize themePrimary.